action/edit.go

```diff
@@ -69,7 +70,10 @@ func (s *Action) editor(content []byte) ([]byte, error) {
 		return []byte{}, fmt.Errorf("failed to create tmpfile to start with %s: %v", editor, tmpfile.Name())
 	}
 
-	cmd := exec.Command(editor, tmpfile.Name())
+	cmdArgs := strings.Split(editor, " ")
```

Comment:
In the `tests` package we use https://godoc.org/github.com/kballard/go-shellquote#Split to do this. We should probably do the same here.